object_manager_txt-record.go

```diff
@@ -45,35 +50,27 @@ func (objMgr *ObjectManager) GetTXTRecord(name string) (*RecordTXT, error) {
 	return &res[0], nil
```

Comment:
It makes sense to consider how to distinguish the absence of an object and a real error during object's retrieval. It should be done above, when analyzing the value of the 'err' variable.

Reply:
Error handling updated to separate no record from a real error similar to A record